app/cmd/job_log.go

```diff
@@ -44,16 +46,24 @@ It'll print the log text of the last build if you don't give the build id.`),
 	Args: cobra.MinimumNArgs(1),
 	Example: `jcli job log <jobName> [buildID]
 jcli job log <jobName> --history 1
-jcli job log <jobName> --watch`,
+jcli job log <jobName> --watch
+jcli job log <jobName> --tail <numberOfLines>`,
 	PreRunE: func(_ *cobra.Command, args []string) (err error) {
-		if len(args) >= 2 && jobLogOption.History == -1 {
+		if len(args) >= 3 && jobLogOption.History == -1 {
 			var history int
 			historyStr := args[1]
 			if history, err = strconv.Atoi(historyStr); err == nil {
 				jobLogOption.History = history
 			} else {
 				err = fmt.Errorf("job history must be a number instead of '%s'", historyStr)
 			}
+			numberOfLinesStr :=args[2]
+			numberOfLines, err:=strconv.Atoi(numberOfLinesStr)
+			if err!=nil||numberOfLines<=0 {
+				err =fmt.Errorf(err.Error(),"lines of job must be a positive integer instead of '%s'",numberOfLinesStr)
+			}else{
+				jobLogOption.NumberOfLines=numberOfLines
+			}
```

Comment:
It's not necessary to parse the value of `tail`. The following code can do that.

`jobLogCmd.Flags().IntVarP(&jobLogOption.NumberOfLines,"tail","t",-1,
		i18n.T("the last number of lines of the log"))`

Reply:
All righty. I've changed that. The lines of 'Current build number' and 'Current build url' are not counted in the 'tail number', but there was a bug about the number of lines to be printed and it has been fixed. Thx.